src/main/java/athleticli/commands/sleep/EditSleepCommand.java

```diff
@@ -43,12 +51,15 @@ public String[] execute(Data data) throws AthletiException {
         if (accessIndex < 0 || accessIndex >= sleepList.size()) {
             throw new AthletiException(Message.ERRORMESSAGE_SLEEP_EDIT_INDEX_OOBE);
         }
+
+        assert accessIndex >= 0 : "Index cannot be less than 0";
+        assert accessIndex < sleepList.size() : "Index cannot be more than size of list";
```

Comment:
I think these two assertions are of little significance, because we have just checked them with `if`.